core/builder.go

```diff
@@ -27,6 +27,10 @@ type BuildCfg struct {
 	// If online is set, the node will have networking enabled
 	Online bool
 
+	// If permament then node should run more expensive processes
+	// that will improve performance in long run
+	Permament bool
```

Comment:
> split to: `Online`, `Offline` and `Ephemeral`

those aren't good distinctions. an ephemeral node may be online and may be offline.